validator/src/commands/mod.rs

```diff
@@ -11,3 +11,29 @@ pub mod set_log_filter;
 pub mod set_public_address;
 pub mod staked_nodes_overrides;
 pub mod wait_for_restart_window;
+
+pub trait FromClapArgMatches {
+    fn from_clap_arg_match(matches: &clap::ArgMatches) -> Self;
```

Comment:
Nice, so the trait + below functions look to allow us to use a common harness.

Do any of the commands have fallible parsing ? If so, should this return a `Result` instead ? Maybe we defer until we find out ?

Reply:
I think we can maybe defer it 🤔

Reply:
works for me